web/app/components/post-item.js

```diff
@@ -708,7 +708,7 @@ function findVimeoID(url) {
  * @returns {string|null}
  */
 function findTweetID(url) {
-    if (url.hostname !== "twitter.com") {
+    if (!["twitter.com", "x.com"].includes(url.hostname)) {
```

Comment:
```suggestion
    if (!["twitter.com", "x.com", "mobile.twitter.com"].includes(url.hostname)) {
```

Adding `mobile.twitter.com` too to fix https://github.com/nakamauwu/nakama/issues/77.